packages/@okta/vuepress-site/guides/sign-into-spa/section08/index.md

```diff
@@ -1,8 +1,11 @@
 ---
 title: Handle the Callback from Okta
 ---
-## Handle the Callback from Okta
 
-After successful authentication, the user is redirected back to your application along with access and ID tokens. See [Define the Login Redirect/Callback](define-the-login-redirect/callback).
+After Okta authenticates a user, they're redirected back to your application via the callback route you [defined earlier]. When Okta redirects back, the URL fragment (the portion after `#`) will contain either tokens for the user, or an error if something went wrong.
```

Comment:
After Okta authenticates a user, they're redirected back to your application via the callback route you [defined earlier]. When Okta redirects back, the URL fragment (the portion after `#`) contains either tokens for the user or an error if something went wrong.